Magiczne-Schody/IChainOfResponsibility.h

```diff
@@ -0,0 +1,18 @@
+/*******************************************************
+* Copyright (C) 2018 Mateusz Urbanek <mateuszurbanek@gmail.com>
+*
+* This file is part of project Magiczne Schody.
+*
+* Project Magiczne Schody can not be copied and/or distributed without the express
+* permission of Malgorzata Gora <gora.malgorzata.b@gmail.com>
+*******************************************************/
+#pragma once
+
+class IChainOfResponsibility
+{
+public:
+	// pure virtual functions providing interface framework.
+	virtual void executeCommand(char eCommand) = 0;
+	virtual void addNextLink(ChainOfResponsibility *l) = 0;
+	virtual void setNextLink(ChainOfResponsibility *l) = 0;
```

Comment:
Mnie się wydawało, że klasa bazowa ma zawierać pointer na następny obiekt, a nie widzę tu czegoś takiego.
Jeśli źle mi się wydaję to z góry przepraszam...